lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -192,7 +193,8 @@ Function Choco-Upgrade
         [bool] $allowprerelease,
         [string] $proxy_url,
         [string] $proxy_username,
-        [string] $proxy_password
+        [string] $proxy_password,
+        [bool] $x86
```

Comment:
Please add a trailing comma always (yes, it should have been done before you too ;-))

Reply:
The last entry should not have the trailing comma or else it fails, just another fun PS-ism to make you angry.

Reply:
Oh ok. I forgot that one in Powershell.